app/assets/stylesheets/grid/_omega.scss

```diff
@@ -79,7 +79,32 @@
     margin-#{$direction}: 0;
   }
 
-  @if type-of($query) == number and unit($query) == "n" {
+  @if type-of($query) == string {
+    $query: str-replace($query, " ", "");
+    $operator: false;
+
+    @if str_index($query, "+") {
+      $operator: "+";
+    } @else if str_index($query, "-") {
+      $operator: "-";
+    }
+
+    @if $operator {
+      $operator-index: str_index($query, $operator);
```

Comment:
Name of function `str_index` should be written in all lowercase letters with hyphens instead of underscores